.travis.yml

```diff
@@ -255,7 +255,7 @@ install:
           fi
           phpenv global $PHP
           rm vendor/composer/package-versions-deprecated -Rf
-          ([[ $deps ]] && cd src/Symfony/Component/HttpFoundation; cp composer.json composer.bak; composer require --dev --no-update mongodb/mongodb ^1.9.0)
+          ([[ $deps ]] && cd src/Symfony/Component/HttpFoundation; cp composer.json composer.bak; composer require --dev --no-update mongodb/mongodb ~1.8.0)
```

Comment:
The PR title says 1.10. 

I think you made a typo somewhere.

Reply:
I tried to install 1.10 first. But that version isn't available through PECL yet. Let's try if not installing `mongodb/mongodb` helps.

Reply:
`mongodb/mongodb ` is the higher-level PHP library. That's not the same package that ext-mongodb providing the lower-level driver, and they have separate versioning.

Reply:
That's https://github.com/mongodb/mongo-php-driver vs https://github.com/mongodb/mongo-php-library

Reply:
1.10 does not exists in any of the 2 packages (no idea where this 1.10 comes from in the PR title then). And the library indeed does not have a 1.9 release either. I think the PR should use `^1.8.0` here (rather than switching to a `~` operator) and that the title should be edited.

Reply:
In the initial draft I tried to install version 1.10 of the PHP extension because of mongodb/mongo-php-library#809. That didn't work out as it's not released yet. So I changed the config to instead limit the installation of `mongodb/mongodb` to 1.8 where we do not need to compile the extension ourselves.

Reply:
by the way, I have updated the PR title accordingly